dist/common/scripts/scylla_util.py

```diff
@@ -1008,11 +1008,15 @@ def __load(self):
         self._cfg.optionxform = str
         self._cfg.readfp(f)
 
+    def __fmt(self, val):
+        need_quotes = any([ch.isspace() for ch in val])
+        return '{}="{}"' if need_quotes else '{}={}'
+
```

Comment:
Better to use shlex.quote, it handles more cases:

```
>>> import shlex
>>> shlex.quote('x')
'x'
>>> shlex.quote('x"')
'\'x"\''
>>> shlex.quote('x y')
"'x y'"
>>> 
```

Reply:
@avikivity it is probably correct that shlex.quote can handle more cases to escape, but what we want to implement on #9149 is, aligning output to distribution default style of sysconfig file.
Which is
 - don't use quotes when parameter does not include space
 - use double quotes when parameter does include space
Something like this:
```
SELINUX=enforcing
```
```
CPUPOWER_START_OPTS="frequency-set -g performance"
```

However, shlex.quote seems like always uses single quote, and don't able to customize it, so it doesn't fit for #9149.